packages/forms/src/model.ts

```diff
@@ -1715,6 +1715,27 @@ export class FormArray extends AbstractControl {
     this._onCollectionChange();
   }
 
+  /**
+   * Move an item from one position to another
+   *
+   * @param from Index of the array item to move
+   * @param to Index of the target item position
+   */
+  moveItem(from: number, to: number): void {
+    this._throwIfControlMissing(from);
+    this._throwIfControlMissing(to);
+
+    const delta = to > from ? 1 : -1;
+    const temp = this.at(from);
```

Comment:
Nit: Naming could be more specific

```suggestion
    const controlToMove = this.at(from);
```